test/e2e/storage/testsuites/volumeperf.go

```diff
@@ -129,18 +129,23 @@ func (t *volumePerformanceTestSuite) DefineTests(driver storageframework.TestDri
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
 	ginkgo.AfterEach(func(ctx context.Context) {
-		ginkgo.By("Closing informer channel")
-		close(l.stopCh)
-		ginkgo.By("Deleting all PVCs")
-		for _, pvc := range l.pvcs {
-			err := e2epv.DeletePersistentVolumeClaim(ctx, l.cs, pvc.Name, pvc.Namespace)
-			framework.ExpectNoError(err)
-			err = e2epv.WaitForPersistentVolumeDeleted(ctx, l.cs, pvc.Spec.VolumeName, 1*time.Second, 5*time.Minute)
+		if l != nil {
+			ginkgo.By("Closing informer channel")
+			close(l.stopCh)
```

Comment:
You may also need a check for `stopCh`. My playground shows that an uninitialized channel will also panic: https://go.dev/play/p/vXGHczE-qi_4